tests/test_utils.py

```diff
@@ -33,7 +33,8 @@ class Demo(db.Model):
         __tablename__ = "demo"
         pk = db.Column(sa.Integer, primary_key=True)
         et = db.Column(
-            EncryptedType(type_in=db.Unicode, key=_secret_key), nullable=False
+            StringEncryptedType(length=255, type_in=db.Unicode, key=_secret_key),
```

Comment:
It should not be an issue, but I cannot find where the `length` params comes from in `StringEncryptedType`. Is it necessary given that it was not there earlier?

Reply:
[The](https://github.com/inveniosoftware/invenio-db/pull/158/commits/afba0be767505b0df9aaebfb310212bf78412e4e) problem is VARCHAR on mysql. Otherwise the tests are not working. Which leads me to a question: how long do we support mysql?

Reply:
I think that we should support MySQL if it does not create great issues, otherwise we might need an ORM layer at all :) There could be organization that are required to use MySQL for example.